requirements.txt

```diff
@@ -13,3 +13,4 @@ pandas
 coverage
 nibabel
 parameterized
+tensorboard
```

Comment:
I think we don't need to install tensorboard anymore as PyTorch already supported it, right?

Reply:
this is still needed in order to actually use the functionality